lib/API/schema.json

```diff
@@ -182,5 +182,8 @@
     "type": [
       "boolean"
     ]
+  },
+  "increment": {
+    "type": "number"
```

Comment:
This is wrong, `increment` will be a `string` which is an environment `key` to be incremented